core/src/components/toolbar/toolbar.scss

```diff
@@ -22,7 +22,10 @@
   --opacity: 1;
 
   @include font-smoothing();
-
+  @include padding-horizontal(
```

Comment:
user's padding-top, padding-end... still need to be in `.toolbar-container`. Only the safe-area should be in the parent container, just like the safe-area-top